src/js/logger.js

```diff
@@ -169,7 +169,7 @@ RiseVision.Common.Logger = (function(utils) {
    *  Public Methods
    */
   function log(tableName, params) {
-    if (!tableName || !params || !params.event || isThrottled(params.event)) {
+    if (!tableName || !params || (params.event && isThrottled(params.event))) {
```

Comment:
@donnapep Won't this allow a falsy (undefined, null, etc) `params.event` value pass by removing `!params.event ||`?